chaosazure/common/compute/command.py

```diff
@@ -0,0 +1,89 @@
+import os
+
+from chaoslib.exceptions import FailedActivity, InterruptExecution
+from logzero import logger
+
+from chaosazure import init_client
+from chaosazure.machine.constants import OS_LINUX, OS_WINDOWS, RES_TYPE_VM
+from chaosazure.vmss.constants import RES_TYPE_VMSS_VM
+
+UNSUPPORTED_WINDOWS_SCRIPTS = ['network_latency', 'burn_io']
+
+
+def prepare_path(machine: dict, path: str):
+    os_type = __get_os_type(machine)
+    if os_type == OS_LINUX:
+        result = "/root/burn" if path is None else path
+    else:
+        result = "C:/burn" if path is None else path
+
+    return result
+
+
+def prepare(machine: dict, script: str):
+    _os_type = __get_os_type(machine)
```

Comment:
I gues underscore in `_os_type` is not required and in other local variables as well